apps/src/dance/songs.js

```diff
@@ -15,7 +15,7 @@ import {fetchSignedCookies} from '../utils';
 export async function getSongManifest(useRestrictedSongs, manifestFilename) {
   if (!manifestFilename || manifestFilename.length === 0) {
     manifestFilename = useRestrictedSongs
-      ? 'songManifest2023_v1.json'
+      ? 'songManifest2023_v3.json'
```

Comment:
Only semi-related, but should we snapshot this file into our repo so that we can track the history of changes to it?

Reply:
+1, as one example, I was looking for this file yesterday!

Reply:
+1 Agreed! I felt very strange not including it in this PR. I'll see if I can add it for the next iteration.

Reply:
As far as I understand, we fetch this file from S3 (and allow for other manifests to be loaded via URL params). Would moving this file into the repo complicate that process? Agreed that it would be good to be able to track changes though!

Reply:
Agreed that we should continue using the existing architecture; it's more just for a long-term reference and a way to correlate changes to that file to changes such as those in this PR.  Open to alternate ways to version track that file.